pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.js

```diff
@@ -110,8 +130,43 @@ router.get('/', function (req, res, next) {
             }
           }
         });
-
-        res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
+        if (res.locals.assessment.group_work) {
+          groupAssessmentHelper.getGroupInfo(
```

Comment:
Not necessarily requesting a change here, just treating this as a learning opportunity! Once you get to this many parameters, it can be slightly nicer to collapse all of these into properties on a single `result` object or the like. e.g.

```ts
groupAssessmentHelper.getGroupInfo('id', 'id', (err, result) => {
  if (ERR(err, callback)) return;

  console.log(result.groupMember);
  console.log(result.permissions);
  // ... and so on
});
```